app/docs/md/learn/concepts/styling/customization.md

```diff
@@ -1,28 +1,45 @@
 ---
-title: Customization
+title: Customizing Styles
 ---
 
-It is possible to completely customize your project's utility classes. To do so you will need to do two things:
+It is possible to completely customize your project's utility classes. There are two ways to do this:
 
-## 1. JSON config file
+## enhance.json
 
-Add a `styleguide.json` (or choose another name) file to the root of your project.
+The `enhance.json` file at the root of your project is designed to act as [a single source of configuration for Enhance plugins](/docs/learn/starter-project/configuration).
```

Comment:
Yup, this will be additive. Enhance Styles will be updated check for both `enhance.json` and a config file specified in `.arc`. Config in `.arc` will win if both exist.

It'll be similar to this control flow that I wrote for `arc-image-plugin`: https://github.com/enhance-dev/arc-image-plugin/pull/7/files#diff-39372e53c36320bb89f4c75d4cc7a205c2a31af3c79fd6e713b47bbc04a4eeb9